SwiftMessages/NSBundle+Utils.swift

```diff
@@ -8,20 +8,31 @@
 
 import Foundation
 
+private class BundleToken {}
+
 extension Bundle {
+    // This is copied method from SPM generated Bundle.module for CocoaPods support
     static func sm_frameworkBundle() -> Bundle {
-        let bundle = Bundle(for: MessageView.self)
-        // Check for Swift Package Manager bundle name
-        if let path = bundle.path(forResource: "SwiftMessages_SwiftMessages", ofType: "bundle") {
-            return Bundle(path: path)!
-        }
-        // Check for CocoaPods or Carthage bundle name
-        else if let path = bundle.path(forResource: "SwiftMessages", ofType: "bundle") {
-            return Bundle(path: path)!
-        }
-        // Just return the app bundle
-        else {
-            return bundle
+
+        let bundleName = "SwiftMessages_SwiftMessages"
```

Comment:
This isn't the right bundle name for CocoaPods, so it won't work as-is

Reply:
Ok, I was skeptical of that too 😅 I'll try if I can modify the original a little less to get the correct bundle in SPM case.